<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Organizer Utility</string>
  </property>
  <widget class="QPushButton" name="btnSelectFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select folder</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="logArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFolderName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Chosen folder location appears here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartCleanUp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Cleanup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Report bugs at : https://github.com/sndpwrites/file-organizer-utility</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
